base/multidimensional.jl

```diff
@@ -489,14 +489,14 @@ end
     for T in indexes.parameters
         T <: CartesianIndex ? (M += length(T)) : (M += 1)
     end
-    index_length_expr = index <: Colon ? Symbol(string("Istride_", N+1)) : :(length(index))
+    index_length_expr = index <: Colon ? Symbol("Istride_", N+1) : :(length(index))
```

Comment:
also best not to at-notify users in commit messages, otherwise every time this gets rebased they will get a notification